ql/src/java/org/apache/hadoop/hive/ql/exec/ReplTxnTask.java

```diff
@@ -53,17 +53,6 @@ public int execute() {
     String tableName = work.getTableName();
     ReplicationSpec replicationSpec = work.getReplicationSpec();
     if ((tableName != null) && (replicationSpec != null)) {
-      Table tbl;
```

Comment:
we dont need the check here?

Reply:
This is checking the replacement info from the table below -
```
        tbl = Hive.get().getTable(work.getDbName(), tableName);
        if (!replicationSpec.allowReplacementInto(tbl.getParameters())) {
```

But we don't track at table level now, so removed it. Earlier logic was if the table is not found in the catch block it use to check at db level
```
 } catch (InvalidTableException e) {
        // In scenarios like import to mm tables, the alloc write id event is generated before create table event.
        try {
          Database database = Hive.get().getDatabase(work.getDbName());
          if (!replicationSpec.allowReplacementInto(database.getParameters())) {
```
So, I have kept it to check always at database level